src/react/account/__tests__/AccountUserList.test.tsx

```diff
@@ -88,7 +97,14 @@ const wrapper = ({ children }: { children: ReactNode}) => {
   const iamClient = new IAMClient('http://testendpoint');
   iamClient.login(params);
   return (
-    <QueryClientProvider client={new QueryClient()}>
+    <QueryClientProvider client={new QueryClient({
```

Comment:
The wrapper should likely be moved to a common utils for test file to be mutualised with other tests.
Actually it is already present in `src/react/utils/test.tsx` bellow, so you would just have to factorize this change there and drop this one.